chatsky/pipeline/service/service.py

```diff
@@ -94,6 +93,16 @@ async def run_component(self, ctx: Context, pipeline: Pipeline) -> None:
         else:
             raise Exception(f"Too many parameters required for service '{self.name}' handler: {handler_params}!")
 
+    @property
+    def computed_name(self) -> str:
+        if callable(self.handler):
```

Comment:
I think this is guaranteed by the type annotation and `BaseModel` inheritance.
Try to get a 100% coverage on tests for pipeline.
We'd like to avoid unnecessary lines that are never reached (and cover others).

Reply:
I saw that TYPE_CHECKING is often/always marked as uncovered code. The internet says adding "#pragma: no cover" to them should fix that, cause it's not supposed to be covered. Is there a reason we shouldn't do that?